dragonfly_doe2/doe/model.py

```diff
@@ -128,7 +131,49 @@ def to_inp(self):
             self.site_bldg_data.to_inp(),
             self.constructions.to_inp(),
             fb.polygons,
-            '\n'.join(pl.to_inp() for pl in self.polygons)
+            '\n'.join(pl.to_inp() for pl in self.polygons),
+            fb.wallParams,
+            fb.fixBldgShade,
+            fb.miscCost,
+            fb.perfCurve,
+            fb.floorNspace,
```

Comment:
At some point, we should change these names to follow our naming style for variables.

```suggestion
            fb.wall_params,
            fb.fix_bldg_shade,
            fb.misc_cost,
            fb.perf_curve,
            fb.floor_nspace,
```

Reply:
I have had that general set of text for a l o n g time! will knock out a good bunch now; and keep at it as is topical till done!